Ambit/Source/Ambit/Mode/TestClasses/MockableGltfExport.h

```diff
@@ -25,24 +25,19 @@
  * To be used only during testing.
  */
 UCLASS()
-class UMockableGltfExport : public UGltfExport
+class UGltfExportMock : public UObject, public IGltfExport
```

Comment:
Do we have a hard requirement that class names should match the file names? I left the current file name because I felt it reads better as a file name. Changing the class name to `XYZMock` makes it more readable from a code point of view